site/data/dynamo.ts

```diff
@@ -34,6 +34,7 @@ const collectDisruptionsData = (
 
     if (!parsedDisruption.success) {
         logger.warn(`Invalid disruption ${disruptionId} in Dynamo`);
+        parsedDisruption.error.errors.forEach((error) => logger.warn(`${disruptionId} - ${error.message}`));
```

Comment:
This could be `logger.warn(flattenZodErrors(parsedDisruption.error));` of we want to log out all the errors

Reply:
Added